backend/repository.ts

```diff
@@ -0,0 +1,93 @@
+import {pool} from "./pool";
+import {v4 as uuid} from "uuid";
+
+export type updateProductJSON = {
+    productName:string,
+    price: number,
+    lat: number,
+    lon: number,
+    photo: string,
+    amount: number,
+    cat: string
+}
+
+export class DatabaseFunctions {
+
+    public static async get_orders(){
+        const [result] = await pool.execute('SELECT * FROM `basket_orders`')
+        return result
+    };
+    public static async update_product(uuid: string, object: updateProductJSON){
+        const [result] =  await pool.execute('UPDATE `main_table` SET `name`=:name, `price`=:price, `lat`=:lat, `lon`=:lon, `photo`=:photo, `amount`=:amount, `cat`=:cat WHERE `uuid`= :uuid', {
+            uuid: uuid,
+            name: object.productName,
+            price: object.price,
+            lat: object.lat,
+            lon: object.lon,
+            photo: object.photo,
+            amount: object.amount,
+            cat: object.cat
+        });
+    };
+
+    public static async item_finder(word:string,
+                             minPrice: number,
+                             maxPrice : number){
+        const [rows] = await pool.execute(
+            'SELECT * FROM `main_table` WHERE `name` LIKE ? AND `price` >= ? AND `price` <= ?',
+            [`%${word}%`, minPrice, maxPrice]

```

Comment:
You probably should be consistent - in other methods you pass object as second argument. There you pass array, which seems inconsistent.